Gulpfile.js

```diff
@@ -92,13 +92,62 @@ gulp.task("build-babel-standalone", cb => {
   );
 });
 
-function webpackBuild() {
-  let version = require("./packages/babel-core/package.json").version;
+const envWebpackPlugins = [
+  new webpack.NormalModuleReplacementPlugin(
+    /\.\/available-plugins/,
+    require.resolve(
+      path.join(
+        __dirname,
+        "./experimental/babel-preset-env-standalone/src/available-plugins"
+      )
+    )
+  ),
+  new webpack.NormalModuleReplacementPlugin(
+    /caniuse-lite\/data\/regions\/.+/,
+    require.resolve(
+      path.join(
+        __dirname,
+        "./experimental/babel-preset-env-standalone/src/caniuse-lite-regions"
+      )
+    )
+  ),
+];
 
+gulp.task("build-babel-preset-env-standalone", cb => {
+  pump(
+    [
+      gulp.src(
+        __dirname + "/experimental/babel-preset-env-standalone/src/index.js"
+      ),
+      webpackBuild({
+        filename: "babel-preset-env.js",
+        library: "babelPresetEnv",
+        plugins: envWebpackPlugins,
+        externals: {
+          "babel-standalone": "Babel",
+        },
+        version: require("./experimental/babel-preset-env/package.json")
+          .version,
+      }),
+      gulp.dest(__dirname + "/experimental/babel-preset-env-standalone"),
+      uglify(),
+      rename({ extname: ".min.js" }),
+      gulp.dest(__dirname + "/experimental/babel-preset-env-standalone"),
```

Comment:
These steps are all identical to the regular babel-standalone build, except with a different path. Please remove the repetition :)